ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Node/CVEsTable.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { gql } from '@apollo/client';
+import { pluralize } from '@patternfly/react-core';
+import { Table, Thead, Tr, Th, Td, ExpandableRowContent, Tbody } from '@patternfly/react-table';
+
+import CvssFormatted from 'Components/CvssFormatted';
+import TbodyUnified from 'Components/TableStateTemplates/TbodyUnified';
+import VulnerabilityFixableIconText from 'Components/PatternFly/IconText/VulnerabilityFixableIconText';
+import { TableUIState } from 'utils/getTableUIState';
+import useSet from 'hooks/useSet';
+
+import VulnerabilitySeverityIconText from 'Components/PatternFly/IconText/VulnerabilitySeverityIconText';
+import {
+    getIsSomeVulnerabilityFixable,
+    getHighestVulnerabilitySeverity,
+} from '../../utils/vulnerabilityUtils';
+import { getNodeEntityPagePath } from '../../utils/searchUtils';
+import NodeComponentsTable, {
+    NodeComponent,
+    nodeComponentFragment,
+} from '../components/NodeComponentsTable';
+
+export const nodeVulnerabilityFragment = gql`
+    ${nodeComponentFragment}
+    fragment NodeVulnerabilityFragment on NodeVulnerability {
+        cve
+        summary
+        cvss
+        scoreVersion
+        nodeComponents(query: $query) {
+            ...NodeComponentFragment
+            nodeVulnerabilities(query: $query) {
+                severity
+                isFixable
+                fixedByVersion
+            }
+        }
+    }
+`;
+
+export type NodeVulnerability = {
+    cve: string;
+    summary: string;
+    cvss: number;
+    scoreVersion: string;
+    nodeComponents: (NodeComponent & {
+        nodeVulnerabilities: {
+            severity: string;
+            isFixable: boolean;
+            fixedByVersion: string;
+        }[];
+    })[];
+};
+
+export type CVEsTableProps = {
+    tableState: TableUIState<NodeVulnerability>;
+};
+
+function CVEsTable({ tableState }: CVEsTableProps) {
```

Comment:
Should we make the `CVEsTable` component names more specific to what it's displaying?

Reply:
Yes I agree, this has been kind of annoying when `Ctrl+P` opening files in VSCode with 3+ components using the same name. I can push this in a follow up.